api/openapi/v2/core-metadata.yaml

```diff
@@ -388,30 +375,46 @@ components:
           type: array
           items:
             type: string
-        addressable:
-          $ref: '#/components/schemas/Addressable'
+        baseAddress:
+          type: string
         adminState:
           type: string
     DeviceServiceResponse:
       allOf:
         - $ref: '#/components/schemas/BaseResponse'
       type: object
       properties:
-        service:
-          $ref: '#/components/schemas/DeviceService'
+        created:
+          type: integer
+        modified:
+          type: integer
+        description:
+          type: string
+        id:
+          type: string
+          format: uuid
+        name:
+          type: string
+        lastConnected:
+          type: integer
+        lastReported:
+          type: integer
+        operatingState:
+          type: string
+        labels:
+          type: array
+          items:
+            type: string
+        baseAddress:
+          type: string
+        adminState:
+          type: string
     ErrorResponse:
       allOf:
         - $ref: '#/components/schemas/BaseResponse'
       description: "A response type for returning a generic error to the caller."
       type: object
-    ConfigRequest:
-      description: "A request associated with the /config endpoint."
-      allOf:
-        - $ref: '#/components/schemas/BaseRequest'
-      type: object
     ConfigResponse:
-      allOf:
-        - $ref: '#/components/schemas/BaseResponse'
```

Comment:
Not sure about removing this... @jpwhitemn we should discuss in Core WG meeting.